### ISSUE TITLE
zsh completion broke

### ISSUE THREAD

**Issue by Mellbourn** [TheLocehiliosan/yadm#71]
After installing a fresh copy of `yadm` on a new macbook pro, I get errors when I use tab completion in `zsh`. Perhaps the new bash completion broke zsh completion (not that it was ever great in zsh, it just completed files. I was hoping that bash completion would work for it)

If I type `yadm boot` and then tab, I get a bell sound and error text is output, making it look like this:
```
~% yadm boot(eval):1: command not found: _bash_complete
(eval):1: command not found: _bash_complete
(eval):1: command not found: _bash_complete
yadm 
```

I have tried doing
```
~% autoload bashcompinit
~% bashcompinit   
```
but that does not help.

I have also tried 
```
~% source /usr/local/opt/yadm/etc/bash_completion.d/yadm.bash_completion
```
but it still does not work. However the error message changes

```
~% yadm boot_git:.:42: no such file or directory:                                                                                                                           17:00:49
_git:.:42: no such file or directory:
_git:.:42: no such file or directory:
yadyadm boot
```

I have installed zsh, bash, bash-completion and yadm using homebrew. These are the versions:
* zsh 5.3.1_1 
* bash 4.4.12 
* bash-completion 1.3
* yadm 1.10.0

Strangely, I have an old yadm installation on a different mac, that has just been updated, and the completion is not broke there (it is just the old time file completion)

**Comment by TheLocehiliosan** [comment]
I'm not a user of `zsh` and unfortunately not familiar with zsh completion. I think the completion you experienced in the past is likely some default completion which merely completes file paths for commands where completion is not available. 

Can you tell me what versions are installed where it is working (meaning zsh/bash/yadm/bash-completions)? The completions script was designed to work with bash, but I did not expect it would interfere with any default completions for other shells. When I first added shell completion to the homebrew recipe, I included an option to leave out completions, but the [homebrew maintainers required me to remove](https://github.com/Homebrew/homebrew-core/pull/13224/files/e2f4a9652f26cb8f9e22c2c2e17069b5e3f111ab) the option. 

Clearly the best fix would be to add zsh completions explicitly, but I'll be looking for some community contribution for that.

**Comment by Mellbourn** [comment]
Ok, thanks for taking the time to analyze the problem.

On my old Macbook, where there is no error when completing `yadm` in `zsh`, I have the exact same versions as mentioned above.

I'm no zsh expert, and the zsh completion configuration is confusing, but if I get the time, I'll try to make the completions for zsh.

**Comment by Mellbourn** [comment]
I have now made a simple zsh completion script. Here is the pull request: https://github.com/TheLocehiliosan/yadm/pull/79

**Comment by TheLocehiliosan** [comment]
This has been released in 1.12.0.
